### PR TITLE
Achydra 733 updates to ac public info pages

### PR DESCRIPTION
**Pull request by ej2432**
This change includes edits to the FAQ, Policies, About, and Developers view pages. Edits were made by Kathryn Pope, Esther Jackson, and Jeremiah Mercurio. Additionally, three new logo files have been created and added for display on the About page.

### CODE REVIEW

**Comment by barmintor** [comment]
Can we edit the first commit here to remove the errant `.bash_profile` file addition?

**Comment by ej2432** [comment]
Yes! It looks like I can just delete the file through GitHub and make a new pull request. Is that the way to move forward? Are there any other changes? Should I also edit the .gitignore?

**Comment by barmintor** [comment]
You shouldn't have to make a new pull request even!

File: .gitignore [hunk 1]
```diff
@@ -49,3 +49,4 @@ coverage/*
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.bash_profile
```
Comment (fcd1) [inline .gitignore:52]: No need to ignore .bash_profile now that file has been removed.

Comment (ej2432) [reply .gitignore:52]: Thanaks, @fcd1 . Should I resolve this? If I attempt to do so through github it seems that I'll then be able to merge the change with master. Not sure if I should be doing that or not.

Comment (fcd1) [reply .gitignore:52]: My personal preference is to make the change locally, commit, and push.

Comment (ej2432) [reply .gitignore]: Aw heck. I did it but added the bash profile back in somehow... dang it. Let me change it.

Comment (ej2432) [reply .gitignore]: Fixed!

Comment (barmintor) [reply .gitignore]: When it's a one line change like this, the reviewer can also make a suggestion in Github - @fcd1 I don't know if you've done that before (Eric and I use that feature a fair amount), but we can walk through it next time something like that comes up.

**Comment by ej2432** [comment]
Hi, @fcd1 . Thanks again for reviewing. Just checking to see if you need any other changes to stick with the planned deployment this week. Thank you!

**Pull request merged**